src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -189,6 +193,27 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
             // default BG color.
             _settings.DefaultBackground(ARGB(0, R, G, B));
         }
+        else if (!_settings.BackgroundImage().empty())
+        {
+            Windows::Foundation::Uri imageUri{ _settings.BackgroundImage() };
+            Media::Imaging::BitmapImage image(imageUri);
+
+			// Note that BitmapImage handles the image load asynchronously,

```

Comment:
There might be tabs here, looks a little too indented

Reply:
Fixed in the latest commit.